test_files/test.rb

```diff
@@ -0,0 +1,46 @@
+require "net/http"
+
+class SomeClass
```

Comment:
We should add some test comments (some single line, some multiline) above all types of entities in this file to make sure they can get picked up by the query.

Reply:
Done :+1: